app/models/shared_account.rb

```diff
@@ -0,0 +1,2 @@
+class SharedAccount < ApplicationRecord
```

Comment:
Google, fb, etc. A central list where they can know all the accounts they've created. I have password_hint on there instead of password, but maybe this field should be encrypted in the db just in case ppl don't take the "hint"?